examples/fashion_conv_dense_test.py

```diff
@@ -107,7 +107,7 @@
 model = ConvNet(
     name='conv_net_test',
     input_networks=None,
-    in_dim=(1, 28, 28),
+    in_dim=[(1, 28, 28)],
```

Comment:
wrapping it in a list should be done behind the scenes

Reply:
Yep, it was just for testing purpose, the Basenetwork internally handles them as list of tuples/int (they get converted internally), which we may revisit once we have functional MultiInputNN.